examples/build.gradle

```diff
@@ -44,7 +44,7 @@ dependencies {
 }
 
 application {
-    mainClassName = System.getProperty("main", "ai.djl.examples.inference.ObjectDetection")
+    mainClassName = System.getProperty("main", "ai.djl.examples.inference.biggan.Generator")
```

Comment:
why do we want to switch this?

Reply:
This was just to run my demo easily with './gradlew run -Dai.djl.default_engine=PyTorch' when I'm in the examples folder.

Reply:
we use this SSD example to verify if new DJL version is working. So probably should revert it